<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_settingsDialog">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_general">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_general">
       <item>
        <layout class="QFormLayout" name="formLayout">
         <property name="leftMargin">
          <number>1</number>
         </property>
         <property name="rightMargin">
          <number>1</number>
         </property>
         <item row="1" column="0">
          <widget class="QLabel" name="label_switchMode">
           <property name="text">
            <string>Switch mode:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="comboBox_switchMode">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <item>
            <property name="text">
             <string>Switch (Default)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Push-To-Talk</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Push-To-Mute</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="comboBox_setMicOnStartup">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <item>
            <property name="text">
             <string>Don't change microphone state</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Set microphone enabled</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Set microphone disabled</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_trayIconStyle">
           <property name="text">
            <string>Tray icon style:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="comboBox_trayIconStyle">
           <item>
            <property name="text">
             <string>Light</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Dark</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_setMicOnStartup">
           <property name="text">
            <string>Microphone state at startup:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_pushDelay">
         <property name="leftMargin">
          <number>1</number>
         </property>
         <property name="rightMargin">
          <number>1</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_pushDelayLabels" stretch="0,0,1">
           <property name="spacing">
            <number>3</number>
           </property>
           <item>
            <widget class="QLabel" name="label_pushDelay">
             <property name="text">
              <string>Push delay:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_pushDelayValue">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_pushDelaySuffix">
             <property name="text">
              <string>ms</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QSlider" name="horizontalSlider_pushDelay">
           <property name="maximum">
            <number>3000</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::NoTicks</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_releaseDelay">
         <property name="leftMargin">
          <number>1</number>
         </property>
         <property name="rightMargin">
          <number>1</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_releaseDelayLabels" stretch="0,0,1">
           <property name="spacing">
            <number>3</number>
           </property>
           <item>
            <widget class="QLabel" name="label_releaseDelay">
             <property name="text">
              <string>Release delay:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_releaseDelayValue">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_releaseDelaySuffix">
             <property name="text">
              <string>ms</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QSlider" name="horizontalSlider_releaseDelay">
           <property name="maximum">
            <number>3000</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::NoTicks</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_notifications">
         <property name="title">
          <string>Notifications</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_selfSwitchesNotifications">
            <item>
             <widget class="QLabel" name="label_selfSwitchesNotifications">
              <property name="text">
               <string>When this app switches microphone:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_selfSwitchesShowNotification">
              <property name="text">
               <string>Show notification</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_selfSwitchesPlaySound">
              <property name="text">
               <string>Play sound</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_otherSwitchesNotifications">
            <item>
             <widget class="QLabel" name="label_otherSwitchesNotifications">
              <property name="text">
               <string>When other app switches microphone:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_otherSwitchesShowNotification">
              <property name="text">
               <string>Show notification</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_otherSwitchesPlaySound">
              <property name="text">
               <string>Play sound</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>84</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_hotkeys">
      <attribute name="title">
       <string>Hotkeys</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_hotkeys">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_hotkeysType">
         <item>
          <widget class="QLabel" name="label_hotkeysType">
           <property name="text">
            <string>Hotkeys type:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_hotkeysType">
           <item>
            <property name="text">
             <string>Switch microphone / Push-To-Talk / Push-To-Mute</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Enable microphone</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Disable microphone</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableView" name="tableView_switchMicHotkeys">
         <property name="dragDropMode">
          <enum>QAbstractItemView::InternalMove</enum>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="verticalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_hotkeysButtons">
         <item>
          <spacer name="horizontalSpacer_hotkeysButtonsLeft">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_addHotkey">
           <property name="text">
            <string>Add new hotkey</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_removeHotkey">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Remove selected hotkey</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_hotkeysButtonsRight">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_dialogButtons">
     <item>
      <spacer name="horizontalSpacer_dialogButtonsLeft">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_ok">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_dialogButtonsRight">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_quit">
       <property name="text">
        <string>Quit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
